demo/midgard-node/src/workers/commit-block-header.ts

```diff
@@ -106,6 +111,62 @@ const successfulSubmissionProgram = (
   txHash: string,
 ): Effect.Effect<WorkerOutput, DatabaseError | FileSystemError, Database> =>
   Effect.gen(function* () {
+    yield* Effect.logInfo(
+      "🔹 Inserting included deposits into ImmutableDB and MempoolLedgerDB",
+    );
+    yield* Effect.all(
+      [
+        batchProgram(
+          Math.floor(BATCH_SIZE / 2),
+          insertedDepositUTxOs.length,
+          "inserting-deposits-to-databases",
+          (startIndex: number, endIndex: number) => {
+            const batchUTxOs = insertedDepositUTxOs.slice(startIndex, endIndex);
+            const currentDate = new Date(); // TODO: perhaps the insertion time is provided somewhere?
+            const ledgerTableBatch: LedgerUtils.EntryWithTimeStamp[] =
+              batchUTxOs.map((utxo) => ({
+                [LedgerUtils.Columns.TX_ID]: Buffer.from(
+                  utxo.input().transaction_id().to_raw_bytes(),
+                ),
+                [LedgerUtils.Columns.OUTREF]: Buffer.from(
+                  utxo.input().to_cbor_bytes(),
+                ),
+                [LedgerUtils.Columns.OUTPUT]: Buffer.from(
+                  utxo.output().to_cbor_bytes(),
+                ),
+                [LedgerUtils.Columns.ADDRESS]: utxo.output().address().to_hex(),
+                [LedgerUtils.Columns.TIMESTAMPTZ]: currentDate,
+              }));
+
+            const txTableBatch: TxTable.EntryWithTimeStamp[] = batchUTxOs.map(
+              (utxo) => ({
+                [TxTable.Columns.TX_ID]: Buffer.from(
+                  utxo.input().transaction_id().to_raw_bytes(),
+                ),
+                [TxTable.Columns.TX]: Buffer.from(utxo.to_cbor_bytes()),
```

Comment:
This column should store a whole transaction CBOR, not just the produced UTxO's.

You'll need to define another conversion function, `depositEventToCmlTransaction`, similar to the one you defined for `CML.TransactionUnspentOutput` (also as described in the issue).

This most likely will be a temporary solution, but it'll do for now.